### PR TITLE
Update tutorials

### PR DESCRIPTION
**Pull request by minjikarin**
Covid 19 Page + prototype page updated

### CODE REVIEW

**Comment by tsolakoua** [comment]
@minjikarin the content is very good, but could you please review and update the grammar and a few typos?

**Comment by minjikarin** [comment]
updated the typos & grammar!

File: docs/examples/prototypes.md [hunk 1]
```diff
@@ -1,5 +1,57 @@
 # Prototypes 
 
-## Available Prototypes
+Would you like to explore the applications that you could build with Amadeus Self Service APIs?
```
Comment (tsolakoua) [inline docs/examples/prototypes.md:3]: Self-Service

File: docs/examples/prototypes.md [hunk 2]
```diff
@@ -1,5 +1,57 @@
 # Prototypes 
 
-## Available Prototypes
+Would you like to explore the applications that you could build with Amadeus Self Service APIs? 
+We have prototypes available in [Amadeus for Developers Github](https://github.com/amadeus4dev).
```
Comment (tsolakoua) [inline docs/examples/prototypes.md:4]: Maybe write Github as GitHub?

File: docs/examples/prototypes.md [hunk 3]
```diff
@@ -1,5 +1,57 @@
...
+
+
+Here on this page, we have listed down our prototypes(examples or Demo) based on **use case** for your convenience.
```
Comment (tsolakoua) [inline docs/examples/prototypes.md:7]: Demo -> demos

File: docs/examples/prototypes.md [hunk 4]
```diff
@@ -1,5 +1,57 @@
...
 
-## Available Prototypes
+Would you like to explore the applications that you could build with Amadeus Self Service APIs? 
+We have prototypes available in [Amadeus for Developers Github](https://github.com/amadeus4dev).
+
+
+Here on this page, we have listed down our prototypes(examples or Demo) based on **use case** for your convenience. 
+
+
+| Use Cases                   | 1A API used                                                                                                | Technology           | Github Link                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```
Comment (tsolakoua) [inline docs/examples/prototypes.md:10]: Maybe external people don't know 1A is Amadeus, so better to say Amadeus API used?

Comment (tsolakoua) [reply docs/examples/prototypes.md:10]: same for Github -> GitHub

File: docs/examples/prototypes.md [hunk 5]
```diff
@@ -1,5 +1,57 @@
...
+| --------------------------- | ---------------------------------------------------------------------------------------------------------- | -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Airport & City autocomplete | Airport & City Search                                                                                      | Node, express, React | [amadeus-airport-city-search-mern]("https://github.com/amadeus4dev/amadeus-airport-city-search-mern")                     |
```
Comment (tsolakoua) [inline docs/examples/prototypes.md:12]: The link is not working

File: docs/examples/prototypes.md [hunk 6]
```diff
@@ -1,5 +1,57 @@
...
+| Flight Search               | Flight Offers Search, Flight Choice Prediction, Trip Purpose Prediction and Airport & City Search          | Python, django       | [amadeus-smart-flight-search-django](https://github.com/amadeus4dev/amadeus-smart-flight-search-django)                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| Flight Search               | Flight Offers Search                                                                                       | PHP, wordpress       | [amadeus-flight-search-wordpress-plugin]("https://github.com/amadeus4dev/amadeus-flight-search-wordpress-plugin") |
```
Comment (tsolakoua) [inline docs/examples/prototypes.md:19]: This link is not working either, must be the double quotes

File: docs/examples/prototypes.md [hunk 7]
```diff
@@ -1,5 +1,57 @@
...
+| Itinarary Management        | Trip Purpose Prediction                                                                                    | Python, django       | [amadeus-trip-purpose-django](https://github.com/amadeus4dev/amadeus-trip-purpose-django)                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| On demand flight status     | On-Demand Flight Status                                                                                    | Python               | [amadeus-async-flight-status]("https://github.com/amadeus4dev/amadeus-async-flight-status")                                |
```
Comment (tsolakoua) [inline docs/examples/prototypes.md:27]: Link not working

File: docs/examples/prototypes.md [hunk 8]
```diff
@@ -1,5 +1,57 @@
...
+
+
+
+
+### Prototypes from community
+
+Also, we have much more prototypes that developers in our community built and shared! 
+
+Explore more in [Amadeus for Developers Github](https://github.com/amadeus4dev) by searching the keyword 'prototype', 'demo' or 'example'.
```
Comment (tsolakoua) [inline docs/examples/prototypes.md:36]: GitHub here as well. 
I would suggest to use `prototype` , `demo`, `example` markdown way of quoting

File: docs/examples/prototypes.md [hunk 9]
```diff
@@ -1,5 +1,57 @@
...
+
+
+
+### Prototypes from community
+
+Also, we have much more prototypes that developers in our community built and shared! 
+
+Explore more in [Amadeus for Developers Github](https://github.com/amadeus4dev) by searching the keyword 'prototype', 'demo' or 'example'. 
+
+below the list are available blog articles related to the External prototypes available in Github.
```
Comment (tsolakoua) [inline docs/examples/prototypes.md:38]: below -> Below, External -> external, Github -> GitHub

File: docs/examples/prototypes.md [hunk 10]
```diff
@@ -1,5 +1,57 @@
 # Prototypes 
 
-## Available Prototypes
+You would like to explore the applications that you can build with Amadeus Self-Service APIs?
```
Comment (tsolakoua) [inline docs/examples/prototypes.md:3]: This line was changed to [this](https://github.com/amadeus4dev/developer-guides/pull/32/commits/260485ec08ebe6ce63bfeaad6f93f30c048c213f#diff-6986952fc0dba8e9ebdb091025c5aaa59fc442113a9dfb056a111e01a3155621R3), but in this commit here I see it revereted

File: docs/examples/prototypes.md [hunk 11]
```diff
@@ -1,5 +1,57 @@
 # Prototypes 
 
-## Available Prototypes
+You would like to explore the applications that you can build with Amadeus Self-Service APIs? 
+we have prototypes available in [GitHub](https://github.com/amadeus4dev).
+
+
+Here in this page, we have listed down our prototypes(examples or demos) based on **use case** for your convinence.
```
Comment (tsolakoua) [inline docs/examples/prototypes.md:7]: I think that one was corrected to, but now it looks like before the word convenience.

File: docs/examples/prototypes.md [hunk 12]
```diff
@@ -1,5 +1,57 @@
...
+
+
+
+### Prototypes from community
+
+Also, we have much more prototypes that developers in our community built and shared! 
+
+Explore more in [GitHub](https://github.com/amadeus4dev) by searching keyword `prototype`, `demo` or `example`. 
+
+Blow list are available blog articles related to the external prototypes available in GitHub.
```
Comment (tsolakoua) [inline docs/examples/prototypes.md:38]: Blow list are available -> Below are listed the available blog articles

File: docs/resources/covid-19.md [hunk 13]
```diff
@@ -1,44 +1,182 @@
 # Covid-19 and Travel Safety 
 
-Access travel Restriction data for over 200 countries
 
-[Keep Travelers informed with Amadeus Travel Restriction API ](https://developers.amadeus.com/blog/-introducing-amadeus-covid-19-travel-restrictions-api)
 
-Powered by [Riskline](https://riskline.com/), the travel restrictions data is updated every hour to so you can keep your travelers informed of the latest developments on their trip. 
+"Can I travel to South Korea? Thailand? or Costa Rica? Do I need to have a PCR test? before the flight? after I arrive? How many days of quarantines are required? Where can I find this information?"
```
Comment (tsolakoua) [inline docs/resources/covid-19.md:5]: quarantines -> quarantine

File: docs/resources/covid-19.md [hunk 14]
```diff
@@ -1,44 +1,182 @@
...
-Access travel Restriction data for over 200 countries
 
-[Keep Travelers informed with Amadeus Travel Restriction API ](https://developers.amadeus.com/blog/-introducing-amadeus-covid-19-travel-restrictions-api)
 
-Powered by [Riskline](https://riskline.com/), the travel restrictions data is updated every hour to so you can keep your travelers informed of the latest developments on their trip. 
+"Can I travel to South Korea? Thailand? or Costa Rica? Do I need to have a PCR test? before the flight? after I arrive? How many days of quarantines are required? Where can I find this information?" 
+
+Don't worry, with Amadeus Self Service APIs, you will be able to have the details information of **current restrictions, Number of covid-19 cases, Entry requirements, and Safety scores, etc** with more details in a searched destinations with one single API.
+
+and even more, **safety information** can be retrieved for the destination you would like to visit so that you will be aware of where you are heading to.
```
Comment (tsolakoua) [inline docs/resources/covid-19.md:9]: and -> And

File: docs/resources/covid-19.md [hunk 15]
```diff
@@ -1,44 +1,182 @@
...
+
+and even more, **safety information** can be retrieved for the destination you would like to visit so that you will be aware of where you are heading to. 
 
 ## Search by an area
-the [Travel Restrictions API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/travel-restrictions) can be searched by a country, city or region. 
- 
+
+There are 2 APIs to achieve the information 
+1. [Amadeus Travel Restrictions API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/travel-restrictions) for covid-19 related information 
+2. [Amadeus Safety Place API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/safe-place) for Safety information
```
Comment (tsolakoua) [inline docs/resources/covid-19.md:15]: Safety -> safety I think would be better

File: docs/resources/covid-19.md [hunk 16]
```diff
@@ -1,44 +1,182 @@
...
+
+[Travel Restrictions API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/travel-restrictions) can be searched **by a country, city, or region**. 
+
 ```bash
-curl https://test.api.amadeus.com/v1/duty-of-care/diseases/covid19-area-report?countryCode=KR&cityCode=SEL
+curl https://test.api.amadeus.com/v1/duty-of-care/diseases/covid19-area-report?countryCode=FR&cityCode=PAR
 ```
-
 !!! information
     Country code(2 letters code) is defined in [ISO 3166](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) and city code(3 letters code) is defined in [IATA](https://www.iata.org/en/publications/directories/code-search/), which you can also use [Amadeus Airport & City Search API](https://developers.amadeus.com/self-service/category/air) to get.
```
Comment (tsolakoua) [inline docs/resources/covid-19.md:23]: Country code -> The country code
I find this part a bit hard to understand, maybe split in 2 sentences?
suggestion: "You can use the [Airport & City Search API](https://developers.amadeus.com/self-service/category/air) to get this information". 
I also think the word "Amadeus" is being used a lot before each API but I don't think it's necessary. It makes sense maybe in a blog article, but in a documentation not really.

File: docs/resources/covid-19.md [hunk 17]
```diff
@@ -1,44 +1,182 @@
...
+curl https://test.api.amadeus.com/v1/safety/safety-rated-locations/Q930400878
+```
+
+!!! information 
+    the id is given by your first query which has been searched by others. the search by id was designed to target a specific area that you want to focus on.
+
+
+
+
+## Get Information from the responses
```
Comment (tsolakoua) [inline docs/resources/covid-19.md:42]: suggestions: responses -> responses
Information -> information

File: docs/resources/covid-19.md [hunk 18]
```diff
@@ -1,44 +1,182 @@
...
+Let's go through some highlights information you get from both Travel Restriction API and Safety Place API.
```
Comment (tsolakoua) [inline docs/resources/covid-19.md:43]: "Let's highlight some information"
Travel Restriction API -> Travel Restrictions API

File: docs/resources/covid-19.md [hunk 19]
```diff
@@ -1,44 +1,182 @@
...
+
+
+
+
+## Get Information from the responses
+Let's go through some highlights information you get from both Travel Restriction API and Safety Place API. 
+
+!!!information
+   Don't forget that Amadeus for Developers provides a `Test Environment` with [limited data collections](https://amadeus4dev.github.io/developer-guides/guides/api-data-collection/). 
+   - the data source of travel restrictions API is from [Riskline](https://riskline.com/), they have been sourced from local governments and media. The quantity of information provided may vary from country to country.
```
Comment (tsolakoua) [inline docs/resources/covid-19.md:47]: Travel Restrictions API 
"they have been sourced from" -> I think the data is singular always, so it should be "the data has been sourced"

File: docs/resources/covid-19.md [hunk 20]
```diff
@@ -1,44 +1,182 @@
...
+        "summary": "<p>Authorities have relied on gathering limits and social distancing measures to contain outbreaks since the start of the pandemic. A three-stage lifting of COVID-19 measures was underway but was paused in late November 2021 as a new spike in COVID-19 cases led to rises in deaths and those with severe symptoms, and threatened hospital beds and ICU availabilities. </p>",
+        "diseaseRiskLevel": "High",
+        "diseaseInfection": {
+            "date": "2022-03-06",
+            "level": "Extreme",
+            "rate": 5088.42
 ```
 
-## Get Travel Restrictions information
+from the response of Safety Place API, you will get safety scores by category.
```
Comment (tsolakoua) [inline docs/resources/covid-19.md:68]: from -> From 
Safety Place API  -> Safe Place API

File: docs/resources/covid-19.md [hunk 21]
```diff
@@ -1,44 +1,182 @@
...
+                "women": 34
+            }
+        }
+```
 
-## Get Local Guidance
+### Get Covid-19 related statistics 
 
-## Get Safety information
+from the response of Travel Restriction API: the result provides Covid-19 statistics such as total cases, active cases, and the current rate of infection per 100,000
```
Comment (tsolakoua) [inline docs/resources/covid-19.md:91]: from -> From
Travel Restriction API -> Travel Restrictions API

File: docs/resources/covid-19.md [hunk 22]
```diff
@@ -1,44 +1,182 @@
...
+            "deaths": 9875,
+            "confirmed": 5822626
+        },
+        "hotspots": "<p>Seoul, Incheon, Gyeonggi province</p>",
+        "dataSources": {
+            "healthDepartementSiteLink": "http://www.kdca.go.kr/index.es?sid=a3",
+            "governmentSiteLink": "http://ncov.mohw.go.kr/en/"
+        },
+```
+and what is the vaccination ratio in this country?
```
Comment (tsolakoua) [inline docs/resources/covid-19.md:105]: and -> And

File: docs/resources/covid-19.md [hunk 23]
```diff
@@ -1,44 +1,182 @@
...
+curl https://test.api.amadeus.com/v1/duty-of-care/diseases/covid19-area-report?countryCode=FR&cityCode=PAR
 ```
-
 !!! information
     Country code(2 letters code) is defined in [ISO 3166](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) and city code(3 letters code) is defined in [IATA](https://www.iata.org/en/publications/directories/code-search/), which you can also use [Amadeus Airport & City Search API](https://developers.amadeus.com/self-service/category/air) to get.
 
 
-## Get Covid-19 statistics 
-the result provides Covid-19 Statstics such as total cases, active cases, and current rate of infection per 100,000 
+[Safety Place API](https://developers.amadeus.com/self-service/category/covid-19-and-travel-safety/api-doc/safe-place) can be searched by a given location with **latitude, longitude and radius**, or **Square information** (North, West, South, and east) or **by id**.
```
Comment (tsolakoua) [inline docs/resources/covid-19.md:26]: Safety Place API -> Safe Place API

File: docs/resources/covid-19.md (hunk 15 above)
Comment (minjikarin) [reply docs/resources/covid-19.md:15]: i found that the name of API is 'Safe Place' not 'Safety Place' so I have changed to 'Safe Place' :)

**Review by tsolakoua — APPROVED** [review]

**Pull request merged**